packages/zowe-explorer-api/package.json

```diff
@@ -37,7 +37,9 @@
     "@zowe/zos-tso-for-zowe-sdk": "8.0.0-next.202407232256",
     "@zowe/zos-uss-for-zowe-sdk": "8.0.0-next.202407232256",
     "@zowe/zosmf-for-zowe-sdk": "8.0.0-next.202407232256",
+    "deep-object-diff": "^1.1.9",
     "mustache": "^4.2.0",
+    "preact": "^10.16.0",
```

Comment:
Do we need `preact` dependency in ZE API package? I don't see it imported anywhere inside the `packages/zowe-explorer-api` folder but am probably missing something 😋

Reply:
I don't believe so - I initially added this, with the plan that the API would have functional components as types for specific properties, but since I did not go that route, the dependency is unnecessary here.

Addressed in upcoming commit 👍